src/components/PlantsV2/index.tsx

```diff
@@ -164,6 +166,15 @@ export default function PlantsDashboardV2(): JSX.Element {
   return (
     <PlantsPrimaryPage
       title={strings.DASHBOARD}
+      text={
+        latestObservation?.completedTime
+          ? (strings.formatString(
+              strings.DASHBOARD_HEADER_TEXT,
+              <b>{strings.formatString(strings.SAMPLE_OF, 2000)}</b>,
+              <>{getShortDate(latestObservation.completedTime, locale.activeLocale)}</>
+            ) as string)
+          : undefined
```

Comment:
guessing `2000` is only for testing...
This construction of sentences pattern is not going to be localization friendly. It will break (joining two parts of translated sentences is not guaranteed to make sense as a full sentence in another language). 
I have brought this up with design before. We don't want to be supporting varying styles across text. Maybe just bolden the number or make the entire sentence bold.

Reply:
Yes, not sure where to get that number yet. 
That's totally true. Thanks for catching this up early. Ok, I will bold just the number for now

Reply:
You can certainly ask design for suggestions :)